packages/manager/apps/pci-cold-archive/src/pages/cold-archive/Archive.page.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  TabsPanel,
+  useProductRegionsAvailability,
+  useProject,
+} from '@ovh-ux/manager-pci-common';
+import {
+  BaseLayout,
+  PciGuidesHeader,
+  RedirectionGuard,
+  useProjectUrl,
+} from '@ovh-ux/manager-react-components';
+import { Suspense, useContext, useEffect, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { Outlet, useLocation, useResolvedPath } from 'react-router-dom';
+import {
+  OdsBreadcrumb,
+  OdsBreadcrumbItem,
+} from '@ovhcloud/ods-components/react';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { ROUTE_PATHS } from '@/routes';
+import { useArchives } from '@/api/hooks/useArchive';
+
+export default function ObjectsPage() {
+  const { t } = useTranslation('cold-archive');
+  const location = useLocation();
+  const hrefProject = useProjectUrl('public-cloud');
+  const { data: project } = useProject();
+  const { ovhSubsidiary } = useContext(ShellContext).environment.getUser();
+  const {
+    data: regions,
+    isPending: isRegionsPending,
+  } = useProductRegionsAvailability(ovhSubsidiary);
+  const { data: allArchives, isPending: isPendingArchive } = useArchives(
+    project.project_id,
+    regions ? regions[0] : '',
```

Comment:
with my comment on useArchives :
```suggestion
    regions?.[0] ,
```